### PR TITLE
Holy new site

### PR DESCRIPTION
**Pull request by holy-devs**

### CODE REVIEW

**Comment by holy-devs** [comment]
Let me know if there are any questions regarding the code and whether you want me to jump on a call for a quick walkthrough?

File: src/js/fpjs-widget.js [hunk 1]
```diff
@@ -11,7 +10,7 @@ const apiToken = process.env.FPJS_API_TOKEN;
 const endpoint = process.env.FPJS_ENDPOINT;
 const region = process.env.FPJS_REGION;
 const config = {
-  ipResolution: "full",
+  ipResolution: 'full',
```
Comment (Lisenish) [inline src/js/fpjs-widget.js:13]: Only formatting changes here, right?

File: webpack.config.js [hunk 2]
```diff
@@ -123,12 +128,13 @@ module.exports = {
...
         path.join(sourceDir, 'views', 'partials', '*.hbs'),
         path.join(sourceDir, 'views', 'partials', 'sections', '*.hbs'),
+        path.join(sourceDir, 'views', 'partials', 'components', '*.hbs'),
       ],
       onBeforeSetup: (Handlebars) => {
         return registerHandlersHelpers(Handlebars);
       },
       onBeforeRender: (Handlebars, data) => {
-        console.log(data);
+        // console.log(data);
```
Comment (Lisenish) [inline webpack.config.js:137]: Leftover comment

File: src/css/index.scss [hunk 3]
```diff
@@ -8,19 +9,16 @@
...
-// @import './vendors/splide/core/splide-core.scss';
-// @import './vendors/gliders';
-
 @import './vendors/sliders';
 @import './vendors/select2';
 @import './custom-select';
 
 @import './generic-styles';
 @import './typography';
+// @import './container';
```
Comment (Lisenish) [inline src/css/index.scss:18]: Leftover comment here and on top of the file

**Review by Lisenish — CHANGES_REQUESTED** [review]
Only minor things
Could you please sync master and resolve conflicts?

File: src/js/fpjs-widget.js (hunk 1 above)
Comment (holy-devs) [reply src/js/fpjs-widget.js:13]: Yeah that's probably my Prettier plugin...

Comment (Lisenish) [reply src/js/fpjs-widget.js:13]: Ok, that's fine

**Review by Lisenish — APPROVED** [review]
LGTM

**Pull request merged**